internal/container/container.go

```diff
@@ -14,22 +14,53 @@
 
 package container
 
-import "errors"
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"os"
+	"os/exec"
+	"strings"
+)
 
-var errNotImplemented = errors.New("not implemented")
+func Generate(ctx context.Context, language, apiRoot, apiPath, output, generatorInput string) error {
+	return runGenerate(apiRoot, fmt.Sprintf("google-cloud-%s", language), apiPath)
+}
+
+func Clean(ctx context.Context, language, repoRoot, apiPath string) error {
+	return runCommand("echo", "clean not implemented")
+}
 
-func Generate() error {
-	return errNotImplemented
+func Build(ctx context.Context, language, repoRoot, apiPath string) error {
+	return runCommand("echo", "build not implemented")
 }
 
-func Clean() error {
-	return errNotImplemented
+func Configure(ctx context.Context, language, apiRoot, apiPath, generatorInput string) error {
+	return runCommand("echo", "configure not implemented")
 }
 
-func Build() error {
-	return errNotImplemented
+const dotnetImageTag = "picard"
+
+func runGenerate(googleapisDir, languageDir, api string) error {
+	args := []string{
+		"run",
+		"-v", fmt.Sprintf("%s:/apis", googleapisDir),
+		"-v", fmt.Sprintf("%s:/output", languageDir),
+		dotnetImageTag,
+		"--command=update",
+		"--api-root=/apis",
+		fmt.Sprintf("--api=%s", api),
```

Comment:
This should be api-path, but only if it's actually specified.
(Or we could make the container contract say that it's *always* specified, and to ignore it if it's empty? I don't much mind either way.)

Reply:
I went with the second for now. We can change it later.